doc/how_to/load_matalb_data.rst

```diff
@@ -0,0 +1,66 @@
+Exporting MATLAB Data to Binary & Loading in SpikeInterface
+===========================================================
+
+In this tutorial, we'll go through the process of exporting your data from MATLAB in a binary format and then loading it using SpikeInterface in Python. Let's break down the steps.
+
+Exporting Data from MATLAB
+--------------------------
+
+First, ensure your data is structured correctly. The data matrix should be organized such that the first dimension corresponds to samples/time and the second dimension to channels.
+
+.. code-block:: matlab
+
+   % Define the size of your data
+   num_samples = 1000;
+   num_channels = 384;
+
+   % Generate random data as an example
+   data = rand(num_samples, num_channels);
+
+   % Write the data to a binary file
+   fileID = fopen('your_data_as_a_binary.bin', 'wb');
+   fwrite(fileID, data, 'double');
+   fclose(fileID);
+
+.. note::
+
+   In a real-world scenario, replace the random data generation with your actual data.
+
+Loading Data in SpikeInterface
+-----------------------------
+
+This should produce a binary file called `your_data_as_a_binary.bin` in your current MATLAB directory.
+You will need the complete path (i.e. its location on your computer) to load it in Python.
+
+Once you have your data in a binary format, you can seamlessly load it into SpikeInterface using the following script:
+
+.. code-block:: python
+
+   from spikeinterface.core.binaryrecordingextractor import BinaryRecordingExtractor
+   from pathlib import Path
+
+   # Define the path to your binary file
+   file_path = Path("/The/Path/To/Your/Data/your_data_as_a_binary.bin")
+
+   # Ensure the file exists
+   assert file_path.is_file()
+
+   # Specify the parameters of your recording
+   sampling_frequency = 30_000.0  # in Hz, adjust as per your matlab dataset
+   num_channels = 384  # adjust as per your matlab dataset
+   dtype = "float64"
```

Comment:
```suggestion
   dtype = "float64"  # equivalent of MATLAB double
```
I would just put it as comment here as well as in the pitfalls.

Reply:
Thanks. Yes, this makes sense.